client/apps/Planning/PlanningListSubNav.tsx

```diff
@@ -191,7 +191,7 @@ class PlanningListSubNavComponent extends React.Component<IProps, IState> {
                     </ButtonGroup>
                     {this.props.activefilter == PLANNING_VIEW.EVENTS ? ' ' : (
                         <div>
-                            {gettext('Assigned to: ')}
+                            <span className="sd-margin-l--1 sd-opacity--75 ">{gettext('Assigned to :')}</span>
```

Comment:
we should keep keep that string `gettext('Assigned to:')` without space before `:` otherwise this will be a new string for transifex

Reply:
yep